src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handlePressedKey);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handlePressedKey);
+  }
+
+  handlePressedKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">
+          {this.state.pressedKey !== ''
```

Comment:
`this.state.pressedKey !== '' ` converts to true and if we have  pressedKey we also always get true. This condition would be enough 👇 
```suggestion
          {this.state.pressedKey
```